### PR TITLE
PR 2122 followup

### PR DESCRIPTION
**Pull request by tnguyen-rh**

### CODE REVIEW

**Comment by tnguyen-rh** [comment]
@mwringe
Initially, this PR was intended solely as a vehicle for style/editing updates.  However, in the process, i discovered a [discrepency](https://github.com/openshift/openshift-docs/pull/2122/files#r66197910) to be resolved.  For this reason, i'm moving status (via title change) to WIP, and adding `tech_review` label.

@tpoitras @bfallonf @adellape @ahardin-rh Peer-review please?

File: install_config/upgrading/manual_upgrades.adoc [hunk 1]
```diff
@@ -911,53 +911,46 @@ Elasticsearch while running this and it will not perform an install.
...
 
 [IMPORTANT]
 ====
-All non-persisted metric values will be lost during the metric `refresh`.
-
-All local changes to the metrics configurations will be overwritten with new configurations meant for the latest install.
-This includes things like number of instances in a replica set.
+The update deletes all non-persisted metric values,
+and overwrites local changes to the metrics configurations,
+e.g., number of instances in a replica set.
```
Comment (ahardin-rh) [inline install_config/upgrading/manual_upgrades.adoc]: Guidance here says to use e.g. sparingly, so maybe spell the abbreviation out: 
https://obriend.fedorapeople.org/WritingStyleGuide/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-E

**Comment by ahardin-rh** [comment]
@tnguyen-rh just one comment from me. Other than that, looks good!  :smiley_cat:

**Comment by tnguyen-rh** [comment]
@ahardin-rh 
Thanks for the review.  I've updated to use "For example...".

@mwringe 
Thanks for clarifying the discrepency.  I have updated to use `MODE=refresh`.

File: install_config/upgrading/manual_upgrades.adoc [hunk 2]
```diff
@@ -911,53 +911,46 @@ Elasticsearch while running this and it will not perform an install.
...
-After this the metrics components should have been restarted and are now using the most recent versions.
-
+<1> In the original deployment command, there was no `MODE=refresh`.
+====
 
 [NOTE]
 ====
-It is normal to see a gap in the graphs during the time when the `refresh` is occuring. This is due to the
-metrics components not running and being available to gather metrics during this time.
+It is normal to see a gap in the graphs during the `refresh` operation.
```
Comment (bfallonf) [inline install_config/upgrading/manual_upgrades.adoc]: "It is" is expletive. 

http://writing.wisc.edu/Handbook/CCS_expletive.html

Maybe:

"Seeing gaps in the graphs during the `refresh` operation is normal."

**Comment by bfallonf** [comment]
@tnguyen-rh Only one comment from me.

Otherwise, do the links need to be xrefs?

Otherwise, LGTM :100:

File: install_config/upgrading/manual_upgrades.adoc (hunk 2 above)
Comment (tnguyen-rh) [inline install_config/upgrading/manual_upgrades.adoc]: Thanks for catching this.  I like your suggestion and after playing around w/ it a bit have managed to tighten it up even more.  WDYT?

**Comment by tnguyen-rh** [comment]
@bfallonf I've converted all `link`s to `xref`s (w/ attendant `s/html/adoc/` and addition of `#` for references w/o id).  WDYT?

**Comment by bfallonf** [comment]
@tnguyen-rh Thumbs up for everything :+1:

**Comment by tnguyen-rh** [comment]
@ahardin-rh @bfallonf Thanks for the reviews.

[rev_history]
|link:../install_config/upgrading/manual_upgrades.html[Upgrading -> Performing Manual Cluster Upgrades]
|Added link:../install_config/upgrading/manual_upgrades.html#manual-upgrading-cluster-metrics[Upgrading Cluster Metrics] section.
%
|link:../install_config/upgrading/automated_upgrades.html[Upgrading -> Performing Automated Cluster Upgrades]
|Added link:../install_config/upgrading/automated_upgrades.html#automated-upgrading-cluster-metrics[Upgrading Cluster Metrics] section.
%

**Pull request merged**